static/local/blackwomenssuffrage/key-figures.md

```diff
@@ -1,83 +1,199 @@
 and junior college provided academic studies (including courses in Black history), religious instruction and technical training in trades ranging from home economics to printing to shoe repair in order to prepare Black women for the workforce. The National Training School advocated for racial uplift through self-help and was unique in that it was staffed, managed and funded by African Americans. Burroughs herself worked as a teacher at the school, and dedicated much of the rest of her life to the education of Black women. Burroughs, along with many members of the Women's Convention, was a staunch supporter of women's voting rights, believing that they were the key to advancement for Blacks and women. Burroughs also believed that the vote could help protect women from abuse. She  was active within a network of influential Black women suffragists that included [Coralie Franklin Cook](/search?q=Coralie+Franklin+Cook), [Anna Julia Cooper](/search?q=Anna+Julia+Cooper) and [Angelina Weld Grimk&eacute;](/search?q=Angelina+Weld+Grimk&eacute;). She contributed articles in support of women's suffrage for the NAACP publication [The Crisis](/search?q=The+Crisis), including writing in the August 1915 issue, "The ballot, wisely used, will bring to her \[the African-American woman\] the respect and protection that she needs...She needs the ballot to reckon with men who place no value upon her virtue, and to mould healthy public sentiment in favor of her own protection." Burroughs was active in the [National Association of Colored Women](/search?q=National+Association+of+Colored+Women) (NACW), helping to galvanize Black women's support for  the 19th Amendment. She also contributed to the [Association for the Study of Negro Life and History](/search?q=Association+for+the+Study+of+Negro+Life+and+History) and, in 1920, created the National Association of Wage Earners (NAWE), an organization that advocated for the rights of domestic workers. In 1924, she helped to found the National League of Republican Colored Women and served as its president, becoming a popular political speaker in the 1920s. In 1928, President Herbert Hoover appointed her to chair a committee on housing for African Americans. Under the auspices of the NAACP, she endorsed boycotts and other types of protest to fight for an end to segregation and discrimination. In her later years, she remained a popular and respected leader and continued to speak out on issues of Black pride and education. She developed a close friendship and working relationship with civil rights leader [Martin Luther King, Jr.](), the son of her friends Martin Luther King, Sr. and his wife Alberta. This relationship was symbolic of the way in which Burroughs' work helped set a foundation for the Civil Rights Movement. Burroughs continued to lead the NBC Women's Auxiliary until her death in 1961. Several years later, the National Training School for Women and Girls that she founded was renamed the Nannie Helen Burroughs School.
```

Comment:
You can add a link to "Nannie Helen Burroughs" `/search?q=Nannie+Burroughs`
Change `/search?q=Anna+Julia+Cooper` to `/search?q=Anna+Cooper`
Martin Luther King Jr. needs his link filled in `/search?q=Martin+Luther+King+Jr`